moveit_setup_assistant/src/widgets/collision_matrix_model.h

```diff
@@ -0,0 +1,80 @@
+/*********************************************************************
```

Comment:
I think of ``tools`` as the etc. folder. or how about ``src/models`` then? I don't see why it belongs in a ``widgets`` folder if that is not what it is.